sources/structs.h

```diff
@@ -1867,6 +1867,7 @@ struct C_const {
     int     MemDebugFlag;          /* Only used when MALLOCDEBUG in tools.c */
     int     OldGCDflag;
     int     WTimeStatsFlag;
+    int     SortReallocateFlag;    /* Controls reallocation of large+small buffer at module end */
```

Comment:
Can you write the meaning of `SortReallocateFlag` (because it takes 0, 1, 2 if I correctly understand; 0: off, 1: on, 2: temporarily on)? This will be like the comment for `CheckpointFlag`, though VSCode fails to display such multiline comments: https://github.com/microsoft/vscode-cpptools/issues/5811.